lambda_function.py

```diff
@@ -0,0 +1,80 @@
+import json
+import urllib.parse
+import boto3
+import logging
+#from botocore.exceptions import ClientErro
+
+print('Loading function') 
+region = 'us-east-1'
+#bucket_name = 'csv-bucket-3534r35d'
+csv_bucket_name = 'bucket-for-converted-csv-objects'
+csv_object_name = 'converted-csv-object'
```

Comment:
Try to set this as environment variables in terraform and refer them in the function code

Reply:
This is done!